### ISSUE TITLE
Blog page search field says "Start typing"

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/nextcloud.com#991]
Should be "Search blog …" instead. Couldn’t find the code here so I guess it’s in Wordpress?

**Comment by jospoortvliet** [comment]
Yeah, this is a wordpress app. It sadly doesn't work very well, I dug around a little but not much I can do

**Comment by MariusBluem** [comment]
> Yeah, this is a wordpress app. It sadly doesn't work very well, I dug around a little but not much I can do

Manually patching the string?

**Comment by jospoortvliet** [comment]
No idea how or where ;-) Feel free to digg in it but I bet you will have to dive into the wordpress plugin code on our live server so... pfff.
